sp/src/game/server/hl2/npc_combine.cpp

```diff
@@ -354,6 +354,10 @@ DEFINE_INPUTFUNC( FIELD_VOID, "DisablePlayerUse", InputDisablePlayerUse ),
 DEFINE_KEYFIELD( m_iCanOrderSurrender, FIELD_INTEGER, "CanOrderSurrender" ),
 DEFINE_INPUTFUNC( FIELD_VOID, "EnableOrderSurrender", InputEnableOrderSurrender ),
 DEFINE_INPUTFUNC( FIELD_VOID, "DisableOrderSurrender", InputDisableOrderSurrender ),
+
+DEFINE_KEYFIELD( m_bDisablePlayerGive, FIELD_BOOLEAN, "DisablePlayerGive" ),
```

Comment:
Per DMs, consider making this a three state variable so we can force player giving on or off regardless of a soldier's commandable status